mteb/tasks/Retrieval/fr/AlloprofRetrieval.py

```diff
@@ -12,13 +12,13 @@ class AlloprofRetrieval(AbsTaskRetrieval):
         name="AlloprofRetrieval",
         description="This dataset was provided by AlloProf, an organisation in Quebec, Canada offering resources and a help forum curated by a large number of teachers to students on all subjects taught from in primary and secondary school",
         reference="https://huggingface.co/datasets/antoinelb7/alloprof",
-        hf_hub_name="mteb/alloprof",
+        hf_hub_name="antoinelb7/alloprof",
```

Comment:
```suggestion
        hf_hub_name="lyon-nlp/alloprof",
```
We used this one for the retrieval, since we preprcessed the original dataset: https://huggingface.co/datasets/lyon-nlp/alloprof